lib/app.js

```diff
@@ -79,9 +79,7 @@ export class Container extends Component {
 }
 
 const warnUrl = execOnce(() => {
-  if (process.env.NODE_ENV !== 'production') {
-    warn(`Warning: the 'url' property is deprecated. https://err.sh/next.js/url-deprecated`)
-  }
+  warn(`Warning: the 'url' property is deprecated. https://err.sh/next.js/url-deprecated`)
```

Comment:
This is totally necessary. We only want to log this in development.

Reply:
[`lib/utils.js`](https://github.com/zeit/next.js/blob/canary/lib/utils.js#L2-L4) (Lines 2-4) handles it, doesn’t it?

Reply:
Also this makes sure we compile it out using webpack, instead of leaving the warn code in production it's compiled away.

Reply:
Was thinking that it handles and can remove unnecessary check. Sorry :)